openapi.yml

```diff
@@ -0,0 +1,130 @@
+# this API specification is draft and subject to change.
+openapi: "3.0.2"
+info:
+  title: CVE Services API
+  description: The CVE services API which supports automation tooling for the CVE Project.
+  version: 0.1.0
+  contact:
+    "name": "CVE Services API"
+    "url": http://cve.mitre.org/api
```

Comment:
Should this use HTTPS? Same with the ToS link below.

Reply:
Good catch! I'll update the links when I push an update with the other endpoints.